stf/data_validation/data_validation.py

```diff
@@ -31,7 +31,7 @@ def clean(data):
     len_original = len(data)
     # TODO Look into this
     # Remove where load is NA # # df.dropna?
-    data = data.loc[np.isnan(data.iloc[:, 0]) != True, :]
+    data = data.loc[np.isnan(data.iloc[:, 0]) != True, :]   # noqa E712
```

Comment:
What does # noqa E712 mean?

Reply:
This make flake8 ignore the E712 violation here. In this case we need to use `!= True` and we can't use ` is not True` which is normally the preferred option.

![image](https://user-images.githubusercontent.com/6715707/108057645-2fb63980-7053-11eb-810a-4a65e76a4062.png)